### PR TITLE
Emorenoo patch 1

### PR DESCRIPTION
**Pull request by emorenoo**

### CODE REVIEW

File: allure-results/64a7cd00-5272-4d46-9757-d631da79a5a1-attachment.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<div>Status code 400 </div>
+
+
+<h4>Body</h4>
+<div>
+    <pre class="preformated-text">
```
Comment (DairoQuintero-KSO) [inline allure-results/64a7cd00-5272-4d46-9757-d631da79a5a1-attachment.html]: Don't include this report files

File: allure-results/673649ff-9ca3-4d3e-921a-5ab43f5fd432-attachment.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        <div>access-control-allow-methods: GET, POST, PUT, PATCH, DELETE, OPTIONS</div>
+        <div>access-control-expose-headers: X-Parse-Job-Status-Id, X-Parse-Push-Status-Id</div>
+        <div>Via: 1.1 b03cdde00266d1a718dac82499df6b8e.cloudfront.net (CloudFront)</div>
+        <div>access-control-allow-origin: *</div>
+        <div>access-control-allow-credentials: true</div>
+        <div>x-content-type-options: nosniff</div>
+        <div>x-xss-protection: 1; mode=block</div>
+        <div>Content-Encoding: gzip</div>
+        <div>x-powered-by: Express</div>
+        <div>X-Amz-Cf-Id: Pgw8Vtdy7hSvdcWkOLfHR8vPycpnjEG2LhiNIlZvavNKITXNLaezoQ==</div>
```
Comment (DairoQuintero-KSO) [inline allure-results/673649ff-9ca3-4d3e-921a-5ab43f5fd432-attachment.html]: Don't include this report files

**Review by DairoQuintero-KSO — CHANGES_REQUESTED** [review]
Ok, delete the report files, don't include them in the PR

File: allure-results/64a7cd00-5272-4d46-9757-d631da79a5a1-attachment.html (hunk 1 above)
Comment (emorenoo) [reply allure-results/64a7cd00-5272-4d46-9757-d631da79a5a1-attachment.html]: I have deleted all reports files.

File: allure-results/673649ff-9ca3-4d3e-921a-5ab43f5fd432-attachment.html (hunk 2 above)
Comment (emorenoo) [reply allure-results/673649ff-9ca3-4d3e-921a-5ab43f5fd432-attachment.html]: I have deleted all reports files.

**Review by DairoQuintero-KSO — APPROVED** [review]
Great!

**Pull request merged**